internal/preferences.go

```diff
@@ -183,7 +184,7 @@ func (p *InMemoryPreferences) StringWithFallback(key, fallback string) string {
 		return fallback
 	}
 
-	return value.(string)
+	return fmt.Sprint(value)
```

Comment:
I don't quite think this is the right approach. It definitely solves the issue but it is a bit brute-force as it will be quite slow and basically cause a copy if there is a string that is passed to it (which we probably expect to be there most of the time). I would rather prefer if we check that the value actually is a string and log an error, or something like that, if it isn't.

Reply:
Hi,
Problem is value.(string) returns a boolean and not the string.  I do not think this is the expected behavior.  Also since it is an interface, there may not be much else that can be done.  I must admit I am new to go, if there is a preferred way to handle this, I will gladly comply.  As far as error handling, that is a good idea.  Ill add tests once we are in agreement about the patch.

Reply:
Erm I think there is a misunderstanding here.
`mything.(string)` will return a string (or panic if mything was not a string or castable). It certainly does not return a bool.

Reply:
P.s. `string` is not an interface, it is the builtin type representing a sequence of `rune`s.